socorro/external/fs/crashstorage.py

```diff
@@ -167,11 +148,12 @@ def _save_files(self, crash_id, files):
                     f.write(contents)
 
     def save_raw_crash(self, raw_crash, dumps, crash_id):
-        if dumps is None:
-            dumps = MemoryDumpsMapping()
+        dumps = dumps or {}
```

Comment:
Previously, this took a different type for `dumps` than other crashstorage. Fun.